docker-compose.yml

```diff
@@ -4,6 +4,7 @@ services:
 
   localstack:
     image: localstack/localstack@sha256:7c6635493185d25165979995fb073fd789c72b6d8b17ef3a70b798d55576732f
+    platform: linux/amd64
```

Comment:
probably not necessary for `localstack`.  if running container on M1, and an arm64 platform image is available, that should be fine to use (and faster, presumably).  in general, a platform only matters for containers we're building ourselves.